frontend/src/component/project/ProjectList/ProjectList.tsx

```diff
@@ -222,42 +207,104 @@ export const ProjectListNew = () => {
+                                            key={project.id}
+                                            name={project.name}
+                                            id={project.id}
+                                            mode={project.mode}
+                                            memberCount={2}
+                                            health={95}
+                                            featureCount={4}
+                                        />
+                                    ))
+                                }
+                                elseShow={() => (
+                                    <>
+                                        {projects.map(
+                                            (project: IProjectCard) => (
+                                                <StyledCardLink
+                                                    key={project.id}
+                                                    to={`/projects/${project.id}`}
+                                                >
+                                                    <ProjectCard
+                                                        onHover={() =>
+                                                            handleHover(
+                                                                project.id,
+                                                            )
+                                                        }
+                                                        name={project.name}
+                                                        mode={project.mode}
+                                                        memberCount={
+                                                            project.memberCount ??
+                                                            0
+                                                        }
+                                                        health={project.health}
+                                                        id={project.id}
+                                                        featureCount={
+                                                            project.featureCount
+                                                        }
+                                                        isFavorite={
+                                                            project.favorite
+                                                        }
+                                                    />
+                                                </StyledCardLink>
+                                            ),
+                                        )}
+                                    </>
+                                )}
+                            />
+                        </StyledItemsContainer>
+                    }
+                />
+            </StyledProjectGroupContainer>
+        );
+    };
```

Comment:
This is just extracting the component that was already there, so that we can use it in two places. However, it feels pretty gnarly. 

I'm unsure about moving it out into its own file, though, because then we'd have to add five extra parameters to the function:
- `loading`
- `searchValue`
- `CardComponent`
- `ContainerComponent`
- `handleHover`

Still, it might be a neater approach. What do you think?

Reply:
Update: I went ahead with extracting this into its own file. I think it's the right thing to do.